packages/kbn-plugin-generator/sao_template/template/public/components/app.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { BrowserRouter as Router, Route, withRouter, RouteComponentProps } from 'react-router-dom';
+
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiPageContentHeader,
+  EuiPageHeader,
+  EuiTitle,
+  EuiText,
+} from '@elastic/eui';
+
+import { AppMountContext, AppMountParameters } from '<%= relRoot %>/../src/core/public';
+
+const FooApp = ({ basename, context }: { basename: string; context: AppMountContext }) => (
+  <Router basename={basename}>
+      <EuiPage>
+        <EuiPageBody>
+          <EuiPageHeader>
+            <EuiTitle size="l">
+              <h1>
+                <FormattedMessage
+                  id="liz.helloWorldText"
+                  defaultMessage="Hello World!"
```

Comment:
NIT: bundle prefix should be something like `snake_case(pluginName)`. Also we might want to display the plugin name in the app component's title.